### ISSUE TITLE
Add ability to change font size in webapp 

### ISSUE THREAD

**Issue by srounce** [sourcegraph/sourcegraph#133]
#### Describe the feature request.
Add ability to change font size in webapp (and save it).

#### Is your feature request related to a problem? Please describe.
I have a 4k monitor, I like browsing repos/reading code with SG but I always have to bump the font size up to 125% to get it readable/not ruin my vision. Problem with this is then other tabs are huge (they don't need to be and bad layouts break) so I'm constantly adjusting browser zoom only when using SG (annoying!).

#### Describe alternatives you've considered.
* Clone repo: Defeats the whole point of me using SG
* Use a separate browser user for SG: Seriously?

#### Additional context
None.

**Comment by francisschmaltz** [comment]
@srounce I'll look into adding this! Currently I'm looking into a few accessibility related features that we should have, this being one of them. 

To help me cover my basis, can you let me know what browser, os, screen, resolution you're using?

Thanks!

**Comment by sqs** [comment]
@francisschmaltz This has come up several times (also in #120), so tentatively prioritizing it for November 2018.

**Comment by francisschmaltz** [comment]
@sqs Much like an IDE, we should allow font size to be changed on a user's config. I think this would be the best way to tackle this.

**Comment by sqs** [comment]
It seems this user thinks our font size is smaller than average. If so, then we could solve the problem by increasing our font size for everyone. (Your solution and the original proposed solution would only apply to a small subset of people who learn that feature exists, so I worry it would only solve a tiny percent of the problem.)

**Comment by francisschmaltz** [comment]
I don't think increasing the font size by default is the right choice. We use the same size for code as GitHub, GitLab.

**Comment by francisschmaltz** [comment]
@srounce can you describe your issue in more detail? What is wrong with how the view is laying out? What browser and OS are you using? 

@sqs if this problem is a problem on Sourcegraph it would be a problem on GitHub and GitLab as well.

**Comment by sqs** [comment]
@srounce Can you please provide additional information?

- What browser and OS are you using?
- Do you encounter this issue on GitHub and GitLab, too? (They both use the same base font size as Sourcegraph.)

**Comment by laubzega** [comment]
If the main purpose of the app is to let people stare at code for hours, why is there even a discussion about font size/type? Given the variety of browsers, monitors, OS-es and - most importantly - individual preferences, you are unlikely to settle on a single choice that even roughly satisfies everyone. Make the damn thing as configurable as possible and let the user decide.

I am also not buying the argument about "a small subset of people who learn that feature exists" - this is not a software for general public, but for SWEs, who - again - are expected to use it to READ CODE. If you think these people's first thought when the font is too small/ugly is not "I wonder if I can change the font", then I have an assortment of bridges to sell you.

**Comment by francisschmaltz** [comment]
Closing. Duplicate of https://github.com/sourcegraph/sourcegraph/issues/120

**Comment by srounce** [comment]
> * What browser and OS are you using?

I think it's more related to screen size/resolution/seating distance. The issue is when I'm on 42" 4k monitors with font scaling at 100% (because some applications don't play nice with it), so I scale windows individually (even different font scaling on different browser tabs). Because of the single-column layout of Github/Gitlab code viewing I'm not as badly affected. The code viewer section will expand until 100% of the window width and then I just have characters displayed per line. In Sourcegraph the sidebar (which I really like) just gets bigger and bigger as I zoom until it takes up most of the window. I don't want to close the sidebar because it's really useful (and nicer to look at than any GH sidebar plugin I've used), I don't *really* want to increase the page fontsize, just wanna increase the **code** size.

> * Do you encounter this issue on GitHub and GitLab, too? (They both use the same base font size as Sourcegraph.)

Nah, their layouts scale well, because single-column.
